content/blog/2025/05/snapd-update/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Update on Snaps
+author: silke
+categories:
+  - news
+date: 2025-05-05
+featuredimage: /snap-banner-image.jpg
+url: /2025/05/05/snapd-update
+---
+
+Last year we announced the [removal of snaps from Solus][1] in January of 2025. Because of recent developments, we have decided to hold off on the complete removal for the time being.
+
+## What has changed
+
+There is progress on getting the required AppArmor support added to the Linux kernel itself. The changes have made it to `linux-next`, which means that they *could* land in the next Linux release.
```

Comment:
> in the next Linux release

Which release would that be? We should be mindful about people reading this blog post well into the future, when the "next release" then may not be the "next release" today.